studio/components/interfaces/Organization/Documents/DPA.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ScaffoldSection,
+  ScaffoldSectionContent,
+  ScaffoldSectionDetail,
+} from 'components/layouts/Scaffold'
+import { Button, IconExternalLink } from 'ui'
+
+const DPA = () => {
+  return (
+    <>
+      <ScaffoldSection>
+        <ScaffoldSectionDetail>
+          <div className="sticky space-y-6 top-12">
```

Comment:
you can use classname on the components here, like `<ScaffoldSectionDetail>`

don't need to keep adding more divs inside